app-nuxtjs-starter/auth.ts

```diff
@@ -0,0 +1,13 @@
+import { AuthHandlerParams } from '@storyblok/app-extension-auth'
+
+export const endpointPrefix = '/server/api/authenticate'
+
+export const authHandlerParams: AuthHandlerParams = {
+  clientId       : process.env.CLIENT_ID ?? "",
```

Comment:
Best not to use default values. If one forget to set this environmental variable and get an error, it will be difficult to debug.